web/src/components/forms/consumption.tsx

```diff
@@ -0,0 +1,193 @@
+  });
+
+  const [transmittedPackets, tpLoaded, tpError] = usePrometheusPoll({
+    endpoint: PrometheusEndpoint.QUERY,
+    query: `sort_desc(sum(irate(container_network_transmit_packets_total{cluster="",namespace=~".+"}[4h])) by (node,namespace,pod))`
+  });
+
+  const getCurrentSampling = React.useCallback(() => {
+    return flowCollector?.spec?.agent?.ebpf?.sampling || 50;
+  }, [flowCollector?.spec?.agent?.ebpf?.sampling]);
+
+  const getSamplings = React.useCallback(() => {
+    const current = getCurrentSampling();
+    let samplings = [1, 25, 50, 100, 125, 150];
+    if (!samplings.includes(current)) {
+      samplings.push(current);
+      samplings = _.sortBy(samplings);
+    }
+    return samplings;
+  }, [getCurrentSampling]);
+
+  const loadingComponent = () => <Spinner size="lg" />;
+
+  const errorComponent = () => <WarningTriangleIcon />;
+
+  const value = (response?: PrometheusResponse) => {
+    if (!response) {
+      return 0;
+    }
+    return _.sumBy(response.data.result, r => Number(r.value![1]));
+  };
+
+  const labelsCount = React.useCallback(
+    (label: string) => {
+      if (!rpLoaded) {
+        return loadingComponent();
+      } else if (rpError) {
+        return errorComponent();
+      } else if (!receivedPackets) {
+        return t('n/a');
+      }
+      return _.uniq(_.map(receivedPackets.data.result, r => r.metric[label])).length;
+    },
+    // eslint-disable-next-line react-hooks/exhaustive-deps
+    [receivedPackets, rpError, rpLoaded]
+  );
+
+  const getEstimation = React.useCallback(
+    (sampling: number) => {
+      // eslint-disable-next-line max-len
+      // taken from https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/network_observability/configuring-network-observability-operators#network-observability-total-resource-usage-table_network_observability
+
+      // TODO: rely on more than nodes here
+      const nodes = labelsCount('node');
+      const estimatedCPU = nodes <= 25 ? -0.0096 * sampling + 1.8296 : -0.1347 * sampling + 12.1247;
+      const estimatedMemory = nodes <= 25 ? -0.1224 * sampling + 22.1224 : -0.4898 * sampling + 87.4898;
+      return {
+        cpu: estimatedCPU > 0 ? estimatedCPU.toFixed(2) : '< 0.1',
+        memory: estimatedMemory > 0 ? estimatedMemory.toFixed(0) : '< 1'
+      };
+    },
+    [labelsCount]
+  );
+
+  const getRecommendations = React.useCallback(() => {
+    // eslint-disable-next-line max-len
+    // taken from https://docs.redhat.com/en/documentation/openshift_container_platform/4.18/html/network_observability/configuring-network-observability-operators
+    const nodes = labelsCount('node');
+    return [
+      {
+        cpu: nodes <= 10 ? 4 : 16,
+        memory: nodes <= 10 ? 16 : 64,
+        lokistackSize: nodes <= 10 ? '1x.extra-small' : nodes <= 25 ? '1x.small' : '1x.medium',
+        kafka: nodes <= 10 ? 'n/a' : nodes <= 25 ? '6 consumers' : '18 consumers'
```

Comment:
```suggestion
        kafka: nodes <= 25 ? '6 consumers' : '18 consumers'
```

we recommend kafka for all production cases